framework/doc/content/source/ics/ScalarConstantIC.md

```diff
@@ -1,32 +1,5 @@
 # ScalarConstantIC
 
-!alert! construction title=Undocumented Class
```

Comment:
Forget this one?

Reply:
there s like nothing to say, class description and parameters cover everything